### DISCUSSION TITLE
Maven build failing for 'admin-ui' module in keycloak 23.0.7

### DISCUSSION THREAD

**Discussion by KishoreKeerthi-2312** [keycloak/keycloak#34242] [Archive]
I'm trying to build Keycloak version 23.0.7 with mvn.
Maven build is failing for "admin-ui" module with "keycloak-js" error. Can someone pls help with the possible root cause and fix.

C:\Ems-Keycloak\js\apps\admin-ui>pnpm build

admin-ui@ build C:\Ems-Keycloak\js\apps\admin-ui
wireit

The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
vite v5.0.2 building for production...
transforming...
src/keycloak.ts(5,25): error TS7009: 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.
✓ 4966 modules transformed.
rendering chunks...

❌ [build] exited with exit code 2.
❌ 1 script failed.
 ELIFECYCLE  Command failed with exit code 1

I have tried to installing "keycloak-js" using "npm" but that dint help.

**Comment by skkohli** [comment]
I ran into same error while using keycloak in front end. the line was when i tried to instantiate a keycloak instance. 
let kc = new Keycloak({});
